notebooks/entities-synonyms/embedding_synonyms.py

```diff
@@ -0,0 +1,55 @@
+from src.utils.preprocess import get_n_word_strings
+from src.utils.helper_embedding import get_embedding_synonyms
+
+from os import getenv
+import pandas as pd
+import csv
+import json
+from tqdm import tqdm
+
+
+DATA_DIR = getenv('DATA_DIR')
+
+df_bow = pd.read_pickle(filepath_or_buffer='data/processed/tf_embeddings.pkl')
+df_tfidf = pd.read_pickle(filepath_or_buffer='data/processed/tfidf_embeddings.pkl')
+with open(DATA_DIR + '/kg_entities.csv', encoding='utf-8') as csv_file:
```

Comment:
I had to find out from you what `kg_entities` is, if it's something that you made a comment explaining how to generate it yourself or where to download it is nice for other people

Reply:
Makes sense, I added it to the PR but that won't be picked up in the future so adding a comment in the code will be better.